tfe/provider.go

```diff
@@ -72,6 +72,42 @@ func (c ConfiguredClient) schemaOrDefaultOrganizationKey(resource *schema.Resour
 	return c.Organization, nil
 }
 
+// tfeVersionGTE will return true if the client is configured with a tfe version
+// greater than or equal to the version parameter. It will also return true if
+// the client is configured against Terraform Cloud.
+//
+// The full version must be provided, i.e v202203-1
+func (c ConfiguredClient) tfeVersionGTE(v string) (bool, error) {
```

Comment:
Might as well make this public visibility

```suggestion
func (c ConfiguredClient) TFEVersionGTE(v string) (bool, error) {
```

Reply:
Will do so, but it's slightly unnecessary given everything lives under the `tfe` pkg and is not intended to be consumed externally.